docs/content/administration/config-cheat-sheet.en-us.md

```diff
@@ -761,6 +761,17 @@ and
 - `SEND_BUFFER_LEN`: **100**: Buffer length of mailing queue. **DEPRECATED** use `LENGTH` in `[queue.mailer]`
 - `SEND_AS_PLAIN_TEXT`: **false**: Send mails only in plain text, without HTML alternative.
 
+## Override Email Headers (`mailer.override_header`)
+
+⚠️ This is empyt by default, use it only if you know what you need it for.
+
+examples would be:
+```ini
+[mailer.override_header]
+Reply-To = test@example.com
+Content-Type = text/html, charset=utf-8
```

Comment:
`,` is incorrect for in-value http header fields, should be `;`

Reply:
well ... yes & no 

I wanted to denonstrate to add multibe values (witch is possible as per lib api) ... but the correct way is one value and use ; ...

I might just have the simple example in the docs ...

Reply:
You can demonstrate with `Reply-To`. It seems `,` is correct for such mail headers.